appengine/sendgrid/README.md

```diff
@@ -12,14 +12,15 @@ For more information about SendGrid, see their
 [ae-docs]: https://cloud.google.com/appengine/docs/java/
 [sample-docs]: https://cloud.google.com/appengine/docs/java/mail/sendgrid
 
-## Setup
+## Setup and deploy
 
 Before you can run or deploy the sample, you will need to do the following:
 
 1. [Create a SendGrid Account](http://sendgrid.com/partner/google). As of
    September 2015, Google users start with 25,000 free emails per month.
 1. Configure your SendGrid settings in the environment variables section in
    [`src/main/webapp/WEB-INF/appengine-web.xml`](src/main/webapp/WEB-INF/appengine-web.xml).
+1. Visit /send/email?to=YOUR-EMAIL-ADDRESS
```

Comment:
The host is either localhost or the deployed app I assume? I guess the reader will be able to figure this out.